swap/htlc.go

```diff
@@ -204,6 +221,50 @@ func swapHTLCScript(cltvExpiry int32, senderHtlcKey,
 	return builder.Script()
 }
 
+// swapHTLCScriptV2 returns the on-chain HTLC V2 witness script.
+//
+// <recvr_key> OP_CHECKSIG OP_NOTIF
+//   OP_DUP OP_HASH160 <HASH160(sender_key)> OP_EQUALVERIFY OP_CHECKSIGVERIFY
+//   <cltv timeout> OP_CHECKLOCKTIMEVERIFY
+// OP_ELSE
+//   OP_SIZE <20> OP_EQUALVERIFY OP_HASH160 <preimage_hash> OP_EQUALVERIFY 1
+//   OP_CHECKSEQUENCEVERIFY
+// OP_ENDIF
+func swapHTLCScriptV2(cltvExpiry int32, senderHtlcKey,
+	receiverHtlcKey [33]byte, swapHash lntypes.Hash) ([]byte, error) {
+
+	builder := txscript.NewScriptBuilder()
+	builder.AddData(receiverHtlcKey[:])
+	builder.AddOp(txscript.OP_CHECKSIG)
+
+	builder.AddOp(txscript.OP_NOTIF)
+
+	builder.AddOp(txscript.OP_DUP)
+	builder.AddOp(txscript.OP_HASH160)
+	builder.AddData(input.Ripemd160H(senderHtlcKey[:]))
+	builder.AddOp(txscript.OP_EQUALVERIFY)
+	builder.AddOp(txscript.OP_CHECKSIGVERIFY)
+
+	builder.AddInt64(int64(cltvExpiry))
+	builder.AddOp(txscript.OP_CHECKLOCKTIMEVERIFY)
+
+	builder.AddOp(txscript.OP_ELSE)
+
+	builder.AddOp(txscript.OP_SIZE)
+	builder.AddInt64(20)
+	builder.AddOp(txscript.OP_EQUALVERIFY)
+	builder.AddOp(txscript.OP_HASH160)
+	builder.AddData(input.Ripemd160H(swapHash[:]))
```

Comment:
update names `swap_hash`, `preimage_hash`, `sender_htlc_key` to be consisent across method and documentation (both for v1 and v2)

Reply:
done